src/estimation/bootstrap.jl

```diff
@@ -0,0 +1,59 @@
+# Should use ensemble problem
+struct Bootstraps
+  populations
+  fits
+  stratify_by
+end
+bootstrap(fpm::FittedPumasModel; kwargs...) = bootstrap(fpm.model, fpm.data, coef(fpm), fpm.approx; kwargs...)
+
+function try_fit(args...)
+  try
+    return fit(args...)
+  catch e
+    return e
+  end
+end
+
+function bootstrap(model::PumasModel, data::Population, coef, method; samples=200, stratify_by=nothing)
+  # FIXME check for time varying covariates
+  # This data preprocessing is not written for speed. Most of it could
+  # be written in one long loop, but we keep it like this for clarity.
+  # Find all individual values of the variable we're stratisfying by
+  n_subject = length(data)
+  if !(stratify_by isa Nothing)
+    strata_values = [getproperty(subject.covariates, stratify_by) for subject in data]
+  else
+    strata_values = fill(1, n_subject)
+  end
+  # Find the unique values ...
+  strata_unique = unique(strata_values)
+  n_strata = length(strata_unique)
+  # Find the number of occurences
+  count_unique = [count(strata_values.==s) for s in strata_unique]
+  # To assign each individual to a stratum by number
+  stratum = map(i->findfirst(x->x==i, strata_unique), strata_values)
+  # Find the indeces for the different types of subjects
+  populations = [reduce(vcat, [sample(data[strata_values .== strata_unique[idx_s]], count_unique[idx_s]) for idx_s = 1:n_strata]) for n = 1:samples]
+
+  # add distributed 
+  # fits = pmap(pop->fit(model, pop, coef, method), populations)
+  if nthreads() == 1
+  	fits = map(pop->try_fit(model, pop, coef, method), populations)
```

Comment:
```suggestion
    fits = map(pop->try_fit(model, pop, coef, method), populations)
```
You'd have to pass `kwargs` here